Formula/eprover.rb

```diff
@@ -2,7 +2,9 @@ class Eprover < Formula
   desc "Theorem prover for full first-order logic with equality"
   homepage "https://eprover.org/"
   url "https://wwwlehre.dhbw-stuttgart.de/~sschulz/WORK/E_DOWNLOAD/V_2.5/E.tgz"
-  sha256 "3a72cb5bcf24899134c84cb6c797c699d8d7ddfad0de7b5b654581bb17b3c814"
+  sha256 "8a53dfb7276c10794c3ce98527cfcf977939769e7a5e6dc2eda9b38be3fc404a"
+  license any_of: ["GPL-2.0-or-later", "LGPL-2.1-or-later"]
+  revision 1
```

Comment:
Did anything change in the tarball? If not, we shouldn't need the revision.

Reply:
I don't have the original tarball so I can't exactly what changed.  The email reply I got from @schulzs just said:
> The official release date on the web site is 2020-07-25, but the git commits for the next days are only minor changes to the manual and the README files. It’s probably been details I noticed after the release (there always are some!) and fixed silently.

So it sounds like at least some documentation may change?  I thought the safest option was to do a revision bump so that the bottle is sure to reflect the "final" version of the 2.5 release